get-gke-credentials/src/util.ts

```diff
@@ -0,0 +1,19 @@
+import * as fs from 'fs';
+import path from 'path';
+import { v4 as uuidv4 } from 'uuid';
+
+/**
+ * Write content to a file
+ *
+ * @param fileContent File content to write.
+ * @returns file path.
+ */
+export async function writeFile(fileContent: string): Promise<string> {
+  const workspace = process.env.GITHUB_WORKSPACE;
+  if (!workspace) {
+    throw new Error('Missing GITHUB_WORKSPACE!');
+  }
+  const kubeConfigPath = path.join(workspace, uuidv4());
+  await fs.writeFileSync(kubeConfigPath, fileContent);
```

Comment:
nit: add try/catch to throw a better/more specific error

Reply:
Done and improved testing to test this func